### PR TITLE
Add end to end test coverage and main entry point for bucket

### PR DESCRIPTION
**Pull request by cintamani**
Depends on: https://github.com/DEFRA/defra-ruby-aws/pull/3 

Add the end-to-end feature test and document the Gem's configuration and usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,9 +20,43 @@ Or install it yourself as:
...
+# config/initializers/defra-ruby-aws.rb
+require "defra-ruby-aws"
+
+DefraRuby::Aws.configure do |config|
+  config.buckets = [{
+    # bucket's name, required
+    name: "defra-ruby-aws",
+    # AWS bucket access credentials, required
+    credentials: {
+      access_key_id: "ACCES_KEY_ID",
```
Comment (irisfaraway) [inline README.md]:
```suggestion
      access_key_id: "ACCESS_KEY_ID",
```

Typo :) Are these the names of env vars? If so I think we should include that in the comment.

File: README.md [hunk 2]
```diff
@@ -20,9 +20,43 @@ Or install it yourself as:
 
     $ gem install defra_ruby_aws
 
+## Configuration
+
+Add a new bucket with:
+
+```
+# config/initializers/defra-ruby-aws.rb
```
Comment (irisfaraway) [inline README.md]:
```suggestion
# config/initializers/defra_ruby_aws.rb
```

To follow the usual Rails file naming scheme.

**Review by irisfaraway — COMMENTED** [review]
Just focusing on the docs for now – a couple questions and suggestions!

File: README.md [hunk 3]
```diff
@@ -20,9 +20,43 @@ Or install it yourself as:
...
+require "defra-ruby-aws"
```
Comment (cintamani) [inline README.md]:
```suggestion
require "defra_ruby/aws"
```

**Review by Cruikshanks — APPROVED** [review]
LGTM, though as @irisfaraway had a first pass at this might just want to check in with her before merging.

**Pull request merged**